hide fence and separator

davidcarlisle opened the pull request:
as agreed in the call of 15-08-2024 this hides the `fence` and `separator`  properties for issue #209

davidcarlisle committed:
hide fence and separator

NSoiffer commented:
You might have gone a little overboard. I see you removed `<code class="attribute">accent</code>,`. in 
```
when they exist, are supported using the Boolean attributes <code class="attribute">fence</code>,
    <code class="attribute">separator</code> and <code class="attribute">accent</code>, which can be used to distinguish these cases.</p>
```
I know that's not in core, but it is still in full so maybe this was too aggressive a removal?

davidcarlisle commented:
@NSoiffer  yes I wondered about that, the paragraph didn't have any normative statement and it didn't seem to read well once I removed two of the three attributes it mentioned so I dropped the whole thing. If we want to say something about `accent` at that point I think it would need a completely new paragraph specifically about that rather than the existing general text about unstated "subtle properties" which only made sense when discussing the longer list, I thought.

NSoiffer commented:
Given that it's not in core, maybe the less said about the default values, the better.
